execution_workflows/Roar/workflow/scripts/Roar.R

```diff
@@ -0,0 +1,109 @@
+
+### Libraries
+
+library(roar)
+library(rtracklayer)
+library(org.Mm.eg.db)
+
+# Input files
+args = commandArgs(trailingOnly=TRUE)
+
+# Input gtf
+gtf <- args[1]
+
+# Input bams
+bam1 <- paste(path=args[2],list.files(path=args[2],pattern="*.bam"),sep="/")
+bam2 <- paste(path=args[3],list.files(path=args[3],pattern="*.bam"),sep="/")
+bams <- c(bam1, bam2)
+# Output prefix
+out.prefix <- args[4]
+
+
+
+
+
+
+# Print out inputs
+print(">>>>> INPUT ARGS ARE:")
+print(paste0("   >> GTF:  ",args[1]))
+print(paste0("   >> BAM1: ",bam1))
+print(paste0("   >> BAM2: ",bam2))
+
+
+
+
+### Create RoarData Object
+
+# Generate RoarDataSet
+print(">>>>> GENERATING ROAR DATA SET")
+rds <- RoarDatasetMultipleAPAFromFiles(bams[1:2], bams[3:4], gtf)
```

Comment:
As with above this should be changed. This hardcodes each condition to have two samples only without checking which condition they arise from, so is only valid when have a 2*2 comparison.

We should also consider having the option to specify what is the 'base' condition. I don't think this is very clear from the sample table/README